setup.cfg

```diff
@@ -25,9 +25,9 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires = 
-    torch >=1.7.0,<1.8
-    torchvision >=0.8.0,<0.9
-    torchtext <0.9 # quickfixes 662
+    torch ==1.7.0
+    torchvision ==0.8.1
+    torchtext ==0.8.0 # quickfixes 662
```

Comment:
let's not pin `torchtext` version, we only need it to be smaller than `0.9`, if the user has `0.7` there is no need to upgrade it

Reply:
torchtext 0.8.1 is not compatible with torch 1.7.0 whereas torchtext 0.8.0 is